README.md

```diff
@@ -1 +1,129 @@
-# WashingBot
```

Comment:
Review this file in a new window (with nice rendering) ;)
I added a lot of stuff since you checked it.